andela/mrm-mobile
# 159360676 Switch Room Amenities keyword to Room Resources

### What does this PR do?
Switches the Room amenities keyword to Room resources in the app.

### Description of Task to be completed.
1. Find all instances or room resources that have been named amenities(Both in code and UI).
2. Switch them to resources.

### How should this be manually tested?
- Follow the project set up in the README.
- Select a Location/Office/Block/Floor/Room.
- Check the room information or select find room.

### What are the relevant pivotal tracker stories?
159360676
Update All Amenities Keyword to Read "Resources"

### Screenshots
![screenshot_1534162244](https://user-images.githubusercontent.com/33759757/44031045-619e72da-9f0b-11e8-97d4-43449bc761bb.png)
![screenshot_1534162231](https://user-images.githubusercontent.com/33759757/44031047-621de2a4-9f0b-11e8-8272-5c33631c3a3f.png)
